docs/reference/server-configuration.md

```diff
@@ -330,6 +330,18 @@ statements.
     the default value you set, the format ignores the setting. For information on which formats
     support wrapping and unwrapping, see the [serialization docs](/reference/serialization).
 
+## ksql.plugins.rocksdb.total.memory
+  
+Sets the total memory, in bytes, to use across all RocksDB instances for all
+queries on a ksqlDB server, if the `ksql.streams.rocksdb.config.setter` plugin
+is enabled. This property has no effect if the plugin isn't enabled.
+ 
+## ksql.plugins.rocksdb.num.background.threads
+
+Sets the number of threads to use in the shared RocksDB thread pool, if the
+`ksql.streams.rocksdb.config.setter` plugin is enabled. This property
```

Comment:
`ksql.streams.rocksdb.config.setter` could be configured to point to other custom RocksDB config setters. These properties specifically apply to `KsqlBoundedMemoryRocksDBConfigSetter`.